pkg/exchange/coinbase/stream_handlers.go

```diff
@@ -242,13 +243,22 @@ func (s *Stream) buildMapForMarketStream(channel2LocalSymbolsMap map[types.Chann
 
 		// binding serial market store
 		for symbol, options := range klineOptionsMap {
-			store := types.NewSerialMarketDataStore(symbol, minInterval, true)
+			klineDriver := klinedriver.NewTickKLineDriver(
+				symbol, time.Second*5,
```

Comment:
minInterval is defined as 1sec. Does 5 seconds here having the same meaning?

Reply:
Oh, `minInterval` was 1 minute not 1 sec.
`minInterval` is initialized as 1 sec and updated to the minimal supported interval on Coinbase in the `init()`.

5 seconds does have similar meaning here, which is the klines emitting frequency for the driver.

Reply:
You just remind me `minInterval` is no longer required.
I'll remove it as well.